EpiserverRedirects/Menu/QueryExtension.cs

```diff
@@ -78,6 +78,11 @@ public static class QueryExtension
                 rules = rules.Where(rr => rr.Priority <= query.Priority);
             }
 
+            if (query.HostId != "1" && !string.IsNullOrEmpty(query.HostId))
```

Comment:
Here's a good reason why we might benefit of making HostDto Id a Guid . We should extract this value to some const so that it's not magic "1" but something like HostDto.AllHostId (or something similar)